frontend/kubecloud/src/components/AdminInvoicesTable.vue

```diff
@@ -14,7 +14,9 @@
           :headers="headers"
           :items="invoices"
           class="admin-table"
-          hide-default-footer
+          :page="page"
+          :items-per-page="itemsPerPage"
+          :footer-props="{ showFirstLastPage: true, itemsPerPageOptions: [5, 10, 20, 50] }"
```

Comment:
what is the need of this ?
https://vuetifyjs.com/en/api/v-data-table/#props-items-per-page-options
we can use the props as mentioned here directly

Reply:
No need to. I removed footer props for simplicity.